src/Components/Shared/src/HotReloadFeature.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.HotReload
+{
+    internal static class HotReloadFeature
+    {
+        /// <summary>
+        /// Gets a value that determines if hot reload is supported. Currently, the <c>Debugger.IsSupported</c> feature switch is used as a proxy for this.
```

Comment:
(nit) you could add a link to [https://github.com/dotnet/runtime/issues/51159](https://github.com/dotnet/runtime/issues/51159) here. That way our future selves will know which issue tracks this.